tools/tscdocgen/main.go

```diff
@@ -948,11 +948,15 @@ func createTypeLabel(t typeDocType, indent int) string {
 			}
 			indent--
 			return fmt.Sprintf("%s%s}", label, strings.Repeat(" ", indent*4))
-		} else if index := decl.IndexSignature; index != nil && len(index.Parameters) == 1 {
-			return fmt.Sprintf("{[%s: %s]: %s}",
-				index.Parameters[0].Name,
-				createTypeLabel(index.Parameters[0].Type, indent),
-				createTypeLabel(index.Type, indent))
+		} else if len(decl.IndexSignatures) == 1 {
+			index := decl.IndexSignatures[0]
+			if len(index.Parameters) == 1 {
+				return fmt.Sprintf("{[%s: %s]: %s}",
+					index.Parameters[0].Name,
+					createTypeLabel(index.Parameters[0].Type, indent),
+					createTypeLabel(index.Type, indent))
+			}
+			return "{ ... }"
```

Comment:
had to fixup typedoc support since it now lists index signatures in an array.  no clue if this is the right ting to do, but it works.